modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -0,0 +1,252 @@
+= Gateway Startup Encryption in Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information required to configure the connection between your Mule application and Anypoint Platform.
+
+== How It Works
+
+You can encrypt your Anypoint Platform credentials (client ID, client secret, and your proxy password) and add those encrypted values, along with your encryption key to your runtime's `wrapper.conf` file.
+
+While the runtime is starting it checks for an encryption key. Then it gets the client ID and client secret values from its `wrapper.conf` file. If some of the values in that file are enclosed within the encryption indicators (`![` and `]`), the runtime uses the encryption key it loaded earlier to decrypt the values and use its original content.
+
+This process can also be used to encrypt and decrypt sensitive values in your API policies.
```

Comment:
Actually, the process described above is not used to encrypt and decrypt policy data. Just by supplying the encryption key the policies and contracts are encrypted, but it's not tied to the client id, secret config.

Reply:
Moved this section to be part of the credentials encryption: bc6547d